src/static/riot/admin/server_status.tag

```diff
@@ -1,39 +1,6 @@
+<!-- Old unused tag, we could pull this back at some point? had a gauge and such -->
```

Comment:
maybe just pull this tag file altogether? can always go find it again in git.

Reply:
pulled! ✔️